nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/ElasticsearchRestProcessor.java

```diff
@@ -111,11 +112,11 @@ default String getQuery(FlowFile input, ProcessContext context, ProcessSession s
         if (context.getProperty(QUERY).isSet()) {
             retVal = context.getProperty(QUERY).evaluateAttributeExpressions(input).getValue();
         } else if (input != null) {
-            ByteArrayOutputStream out = new ByteArrayOutputStream();
+            final ByteArrayOutputStream out = new ByteArrayOutputStream();
             session.exportTo(input, out);
             out.close();
 
-            retVal = new String(out.toByteArray());
+            retVal = out.toString();
```

Comment:
I considered this but wasn't sure whether it's best to leave it using the system default (in case someone is using a non-UTF8 Charset) - maybe the best thing really would be to allow users to specify an output (and input?) Charset, but that seems like it should be a separate Jira/PR

I'll look again at this and consider whether UTF8 here would make sense for now